test/powershell/Language/Scripting/Scripting.Followup.Tests.ps1

```diff
@@ -90,4 +90,43 @@ Describe "Scripting.Followup.Tests" -Tags "CI" {
 
         TestFunc2 | Should -Be @("1", "2")
     }
+
+    It "'[NullString]::Value' should be treated as string type when resolving .NET method" {
+
+        $testType = 'NullStringTest' -as [type]
+        if (-not $testType) {
+            Add-Type -TypeDefinition @"
```

Comment:
Suggestion: if you're not using string interpolation/substitution, you should use single quoted strings to show that it is a pure literal

Reply:
Updated using single quote.